nixos/modules/services/misc/nix-daemon.nix

```diff
@@ -117,7 +117,7 @@ in
 
       buildCores = mkOption {
         type = types.int;
-        default = 1;
+        default = 0;
         example = 64;
         description = ''
           This option defines the maximum number of concurrent tasks during
```

Comment:
> "The default is 1."

Pleas update this.

Reply:
Dropped it since, it was redundant with the displayed default value.